### ISSUE TITLE
Planning the next release

### ISSUE THREAD

**Issue by mxamin** [xmlsec/python-xmlsec#315]
Let's discuss the next release :)

@jonathangreen
@jimjag

**Comment by mxamin** [comment]
So far these are the two important issues that need to be handled:
https://github.com/xmlsec/python-xmlsec/issues/314
https://github.com/xmlsec/python-xmlsec/issues/316

**Comment by jonathangreen** [comment]
Agreed @mxamin, I think those are the most important as well. I think distributing wheels for Linux #316 will make #314 less important because fewer people will be compiling themselves, but it would still be nice to handle #314 if we can.

**Comment by mxamin** [comment]
> Agreed @mxamin, I think those are the most important as well. I think distributing wheels for Linux #316 will make #314 less important because fewer people will be compiling themselves, but it would still be nice to handle #314 if we can.

@jonathangreen  I agree, I uploaded the linux wheels in the release note. Can you check that for the reported cases using the wheels would actually solve the problem?

**Comment by jonathangreen** [comment]
It doesn't have to be for next release, the issue has been open for a while, but I think we should look into #239 in the next couple releases. 

That issue was written by the maintainer of lxml so the suggestion of how we should be calling the lxml api should be taken seriously.

**Comment by jonathangreen** [comment]
> @jonathangreen I agree, I uploaded the linux wheels in the release note. Can you check that for the reported cases using the wheels would actually solve the problem?

I made a small workflow [here](https://github.com/xmlsec/python-xmlsec/pull/319) to test our built wheels for linux, osx and windows against the lxml wheels by running our tests. It shows that the wheels work for lxml >= 5.0. From what I can see in the bug reports that included their lxml version they it seems installing wheels would fix issues for all those reports.

**Comment by mxamin** [comment]
Great, then I'll upload the rest of wheels.

**Comment by ryanhiebert** [comment]
If you don’t mind my asking, does this mean that if I install binary versions for the latest version of lxml and this package that I should be good?

**Comment by jonathangreen** [comment]
@ryanhiebert yes that is correct! That is what I expect based on my testing anyway.

**Comment by mxamin** [comment]
FYI, I'm gonna have limited/no access up until May 11th :)

**Comment by antoniomika** [comment]
It seems there are inconsistencies now for people that solved an issue previously between `lxml` and `xmlsec` by installing `lxml` without binaries and expecting to use system installed libraries. There are clearly binaries for `xmlsec` now available on PyPi, which has broken installs. The README mentions needing to install system libraries for Linux, but this is not the case anymore (although, other issues on GitHub mention there are no binary wheels available for `xmlsec`).

What is the desired case now? Will `xmlsec` always ship binaries that are compatible with `lxml`? If not, is the expectation to use `--no-binary` for both `lxml` and `xmlsec`?

Happy for either, I'd just like to make sure this is documented for others!

**Comment by jimjag** [comment]
This works for me. If desired, I can submit this one to PyPI to help w/ the bus factor

**Comment by jimjag** [comment]
New release done for Python 3.13 compatibility. Someone needs to update PyPi (I don't have access/perms)
